src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx

```diff
@@ -71,23 +71,65 @@ describe('components/PDOKAutoSuggest', () => {
     it('should call fetch with municipality', async () => {
       await renderAndSearch('Dam', { municipality: 'amsterdam' })
       expect(fetch).toHaveBeenCalledWith(
-        expect.stringContaining(`${municipalityQs}("amsterdam")`),
+        expect.stringContaining(`${municipalityQs}(amsterdam)`),
         expect.objectContaining({ method: 'GET' })
       )
     })
 
     it('should work with an array for municipality', async () => {
       await renderAndSearch('Dam', { municipality: ['utrecht', 'amsterdam'] })
       expect(fetch).toHaveBeenCalledWith(
-        expect.stringContaining(`${municipalityQs}("utrecht" "amsterdam")`),
+        expect.stringContaining(`${municipalityQs}(utrecht amsterdam)`),
         expect.objectContaining({ method: 'GET' })
       )
     })
 
-    it('should not put quotes around a value starting with "-"', async () => {
-      await renderAndSearch('Dam', { municipality: ['utrecht', '-amsterdam'] })
+    it('should work with quotes', async () => {
+      await renderAndSearch('Dam', {
+        municipality: ['"den bosch"', '-amsterdam'],
```

Comment:
This looks pretty error prone, having to wrap any string with spaces with double quotes. Can't we have this handled by the `PDOKAutoSuggest` component logic?

Reply:
This way you have the freedom to compose this part of the query string as needed. Otherwise you get magic in the code, like we had before.

Reply:
True, but this construct requires extra documentation to explain how to handle name that have spaces.

Reply:
Yes, because it also allows for instance a dash, to ignore a search term.
All magic in the config, or all magic in the code, I would say. Half half would be even more confusing.

To keep all magic in the code, we could introduce an optional second config parameter, like `map.municipalityExclude`.

Then you can do something like:

```
"map": {
  "municipality": "groningen",
  "municipalityExclude": "midden"
}
```

To get a query string like:

```
fq=gemeentenaam:("groningen" -"midden")
```

In the code we'll put double quotes around every term, like before. And put a dash in front of each term to exclude, putting those at the end.

Reply:
Another option, to keep all magic and flexibility in the config, is to only allow a string, not an array, to make it more clear that terms with spaces need quotes.

Like I explained above https://github.com/Amsterdam/signals-frontend/pull/2059#issuecomment-1057830548

Reply:
:thinking: Might indeed be better to use a string instead of an array, since the PDOK service also uses that. Then our implementation is in sync with what PDOK is offering and we don't have to document on top of what PDOK is already doing.

Reply:
Yes, exactly. I think that would be the cleanest approach.